dashboard/vendor/assets/javascripts/jquery.simulate.js

```diff
@@ -324,7 +324,7 @@ $.extend( $.simulate.prototype, {
       moves = options.moves || 3;
 
     // Note that this is not an ideal simulation since we are not generating all the
-    // "legacy" mouse events, as described in #1 at 
+    // "legacy" mouse events, as described in #1 at
```

Comment:
this file is still showing up in the diff. looks like a trailing newline has been removed.